autofill/autofill-impl/src/main/res/layout/item_row_search_no_results.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<com.google.android.material.textview.MaterialTextView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I copied it in from bookmarks or downloads (the other places we have the `SearchBar` and use this pattern). It probably should be `DaxTextView`, but then they all should be or none should be. 

I think i'd rather keep it consistent with the other ones and change them all.

Reply:
This view is separate from the SearchBar though. My thinking is that since ADS is in place already, any new component should use ADS components unless we really can't.

I think for now it is ok that we don't migrate this since the whole Autofill feature haven't been migrated to ADS yet and this could be addressed once they do that later.

Reply:
👍